index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Victor's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Inter">
+</head>
+<body>
+    <header class="header">
+        <a href="#">Welcome</a>
```

Comment:
[ Optional Change ] This change won't stop your project from being approved, but it will improve your work.

- [ ] It would be nice you use your name as the logo of the site. This website is going to serve as your portfolio site going forward and you wouldn't want the logo to be "Welcome" right? 😂. Please use your name instead. 👍

Reply:
Wow, this advice is more than welcome. Thanks. I will do that.